arch/Kconfig

```diff
@@ -801,6 +801,28 @@ config ARCH_KMAP_NPAGES
 		maximum number of pages per region, and the configured size of
 		each page.
 
+config ARCH_UDEV_MAPPING
+	bool "Support mapping devices memory into user space"
```

Comment:
can we reuse share memory virtual space?

Reply:
It seems that shared memory are different. For example,  they are backed by physical allocations and they use GRAN allocation in user space which is size limited. By having independent regions, the two features can be independently selected.